content/en/docs/quickstart-guides/quickstart-sign-non-image-artifact.md

```diff
@@ -0,0 +1,238 @@
+---
+title: "Quickstart: Sign and validate a non-image artifact"
+description: "Quickly set up an OCI-based registry and use notation to sign and validate an SBOM for a container image"
+type: docs
+weight: 2
+---
+
+## Prerequisites
+
+Before you begin, you need:
+
+* Docker installed and running, such as [Docker desktop](https://www.docker.com/products/docker-desktop/).
+* [Notation CLI]({{< ref "/docs/user-guides/installation/cli" >}}) installed and configured.
+
+## Create an OCI-compatible registry
+
+Create and run an OCI-compatible registry on your development computer using the [distribution/distribution](https://github.com/distribution/distribution) with the [image deletion](https://docs.docker.com/registry/spec/api/#deleting-an-image) enabled. The following command creates a registry that is accessible at `localhost:5001`. 
+
+```console
+docker run -d -p 5001:5000 -e REGISTRY_STORAGE_DELETE_ENABLED=true --name registry registry
+```
+
+{{% alert title="Note" color="primary" %}}
+If the host port 5001 is already in use, you can use another host port. 
+{{% /alert %}}
+
+If you want to use Notation with other registries, refer to [which registries are compatible with Notary]({{< ref "/docs/faq#what-registries-are-compatible-with-notary" >}}) for more alternatives. See [Authenticate with OCI-compliant registries]({{< ref "/docs/user-guides/how-to/registry-authentication" >}}) when you log in to another OCI registry.
+
+## Add an image to the OCI-compatible registry
+
+The following commands build and push the [wabbit-networks/net-monitor](https://github.com/wabbit-networks/net-monitor#main) container image to your container registry.
+
+```console
+docker build -t localhost:5001/net-monitor:v1 https://github.com/wabbit-networks/net-monitor.git#main
+docker push localhost:5001/net-monitor:v1
+```
+
+Get the digest value of the *localhost:5001/net-monitor:v1* image using `docker inspect`. For example:
+
+```console
+docker inspect --format='{{index .RepoDigests 0}}' localhost:5001/net-monitor:v1
```

Comment:
@JeyJeyGao Would you mind confirming whether we can use this method to get the right digest?

Reply:
For context, i get the following on my development computer:

```
$ docker inspect --format='{{index .RepoDigests 0}}' localhost:5001/net-monitor:v1
localhost:5001/net-monitor@sha256:68e4c15241bf70845a08b0b864ffea81a388ba9c0f43c400222ff52213f45f0e
```

Reply:
@zr-msft the digest may not be the one that identifies image stored in a remote registry. It could be the digest identifying a local image, so I need Junjie's help to confirm it.

Reply:
@yizha1 The digest is accurate if you build the image locally and **push** it to the registry. However, the docker inspect command only shows the digest of local images, which may not be consistent with that of the registry. In other words, if another user updates the content of the reference but you haven't pulled the image, you may get an incorrect digest.

Reply:
If you build the image, but didn't push to the registry, the value will be empty.

Reply:
Thanks @JeyJeyGao, @zr-msft I think it is OK for a quick start, and I don't have any concern.